bot/cogs/cyber.py

```diff
@@ -121,6 +134,68 @@ def __init__(self, bot: Bot):
 
             await ctx.send(embed=embed)
 
+    @command()
+    async def flag(self, ctx: Context, base: str, level_num: int, challenge_num: int = 0):
+        """
+        Gets information about a specific CyberStart Game level and challenge.
+        If the date is before the start date of game (15th January 2019) it will redirect to game() instead
+        """
+
+        if datetime.date.today() < datetime.date(2019, 1, 15):
+            await self.game.callback(self, ctx)
+            return
+
+        # Gather data from CyberStart Game.
+        with open("bot/data/game.json") as f:
```

Comment:
I really don't think we need to load the titles/descriptions or check that the levels are correct for this meme command.
Could you rewrite this so it only includes the relevant code? (lines 181 - 197)

Reply:
I agree with this, and as such shall change my review to concur.